specification/hybridconnectivity/HybridConnectivity.Management/EndpointResource.tsp

```diff
@@ -23,6 +23,39 @@ model EndpointResource
   >;
 }
 
+/**
+ * @dev A long-running resource action.
+ * @template Resource The resource being acted upon
+ * @template Request The request model for the action
+ * @template Response The response model for the action
+ * @template BaseParameters Optional. Allows overriding the parameters for the operation
+ * @template Parameters Optional. Additional parameters after the path parameters
+ * @template Error Optional. The error response, if non-standard.
+ */
+#suppress "@azure-tools/typespec-azure-core/documentation-required" "template"
+#suppress "@azure-tools/typespec-azure-resource-manager/arm-resource-operation" "template"
+@returnsDoc("Azure operation completed successfully.")
+@autoRoute
+@armResourceAction(Resource)
+@post
+op ArmResourceActionSync<
```

Comment:
Should we rename it to ArmResourceActionSyncWithOptionalBody to reduce misleading with the original one?

Reply:
No, they are in the different namespaces. We should avoid inventing new names which would cause name conflicts easily.